pennylane/transforms/adjoint.py

```diff
@@ -113,11 +113,11 @@ def circuit():
     @wraps(fn)
     def wrapper(*args, **kwargs):
         with stop_recording(), QuantumTape() as tape:
-            fn(*args, **kwargs)
+            res = fn(*args, **kwargs)

```

Comment:
this is the fix I made, right? 😆 I can't recall what it happening here any more.

Reply:
Me neither...